MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.RecyclerView/MvvmCross.Droid.Support.V7.RecyclerView.nuspec

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
fixed in https://github.com/MvvmCross/MvvmCross/pull/1568/commits/96b21f303d65024ce833287a7ffc5fec635028e0